CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-rc.5 July 28, 2021
+
+**The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
+
+## Breaking changes ⚠️
+* Improve camera API consumption from java programming language by adding a JvmName annotations. ([#495](https://github.com/mapbox/mapbox-maps-android/pull/495))
```

Comment:
Can we be a bit more specific here? or should we be more specific in release notes?

Reply:
I think we should be both specific and concise here, and maybe we can expand further in the release notes.